pkg/k8s/types.go

```diff
@@ -73,8 +78,35 @@ const (
 	Retry InformerKey = "Retry"
 	// UpstreamTrafficSetting lookup identifier
 	UpstreamTrafficSetting InformerKey = "UpstreamTrafficSetting"
+	// TrafficSplit lookup identifier
+	TrafficSplit InformerKey = "TrafficSplit"
+	// HTTPRouteGroup lookup identifier
+	HTTPRouteGroup InformerKey = "HTTPRouteGroup"
+	// TCPRoute lookup identifier
+	TCPRoute InformerKey = "TCPRoute"
+	// TrafficTarget lookup identifier
+	TrafficTarget InformerKey = "TrafficTarget"
 )
 
+// TrafficTargetListOpt specifies the options used to filter TrafficTarget objects as a part of its lister
+type TrafficTargetListOpt struct {
```

Comment:
I think these are defined in multiple places, and should be removed from k8s package?